draft-06/yang/ietf-netconf-private-candidate@2024-09-12.yang

```diff
@@ -0,0 +1,128 @@
+module ietf-netconf-private-candidate {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:ietf-netconf-private-candidate";
+  prefix pc;
+
+  import ietf-nmda-compare {
+    prefix cmp;
+  }
+
+  organization
+    "IETF NETCONF (Network Configuration) Working Group";
+
+  contact
+    "WG Web:   <http://tools.ietf.org/wg/netconf/>
+     WG List:  <netconf@ietf.org>
+
+     WG Chair: Kent Watsen
```

Comment:
WG Chair information can be omitted

https://datatracker.ietf.org/doc/html/rfc8407#section-4.8